### PR TITLE
Add d language example to bubble sort

### PR DESCRIPTION
**Pull request by scarface-one**
This will add an example implementation of bubble sorting in d, based on the C++ implementation.

### CODE REVIEW

File: book.json [hunk 1]
```diff
@@ -79,6 +79,10 @@
 					"lang": "LabVIEW",
 					"name": "LabVIEW"
 				}
```
Comment (Butt4cak3) [inline book.json]: You have to add a comma after this `}` or it's not valid JSON.

Comment (scarface-one) [reply book.json]: fixed, thanks!

**Comment by Butt4cak3** [comment]
I don't think we have any D programmers here who could properly review your pull request, but I will take a look at it later today (I'm in Europe) and see what I can do. Bubble Sort is not that complicated.

**Comment by scarface-one** [comment]
Thats fine by me. You can find info on the D language here : https://dlang.org/ , and the documentation for the standard library is here : https://dlang.org/phobos/index.html . To build and the file, you can use `dmd bubble_sort.d && ./bubble_sort`, or to make it easier, `rdmd bubble_sort.d`

**Pull request merged**